cube/CMakeLists.txt

```diff
@@ -99,10 +99,10 @@ elseif(CMAKE_SYSTEM_NAME STREQUAL "Linux")
             message( FATAL_ERROR "Selected XLIB for cube build but not building Xlib support" )
         endif()
         set(CUBE_INCLUDE_DIRS
-            ${X11_INCLUDE_DIR}
```

Comment:
This needs to be reverted and put back the way it was.   This is the X11 (Xlib) path.  Not sure why you are pulling in XCB stuff.